.github/workflows/conformance.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         - pubsub.natsstreaming
         - pubsub.kafka
         - pubsub.pulsar
+        - pubsub.mqtt
```

Comment:
Can we have mqtt-<implementation>? And then add multiple mqtts in the test matrix?

Reply:
We could. We'd have to make sure each broker is on its own unique port via Docker Compose.